### PR TITLE
revised procedure to start portal and mobile apps

### PR DESCRIPTION
**Pull request by oraNod**
suggested changes to procedures for starting the portal and mobile apps. start the portal app first. also provide URLs for applications.

### CODE REVIEW

File: docs/workforce-management-framework/topics/pro-running-the-demo-repositories.adoc [hunk 1]
```diff
@@ -67,16 +67,29 @@ NOTE: The terminal is out of use while the Core Server is running.
...
+1&#46; Start the Portal application as follows:
 
 [source,bash]
 ----
+cd raincatcher-portal/
 npm run start
 ----
 
-Both the Mobile and Portal Applications open automatically in a browser.
+2&#46; Start the Mobile application as follows:
```
Comment (wtrocki) [inline docs/workforce-management-framework/topics/pro-running-the-demo-repositories.adoc]: I'm personally not fan of unicode format, but I do nto want to block you here.
what is that? Does it render properly?

File: docs/workforce-management-framework/topics/pro-running-the-demo-repositories.adoc [hunk 2]
```diff
@@ -67,16 +67,29 @@ NOTE: The terminal is out of use while the Core Server is running.
...
+----
+cd raincatcher-mobile/
+npm run start
+----
+
+The Portal and Mobile applications automatically open in a browser at the
+following URLs:
+
+* Portal application: `http://localhost:8001/cookie-login`
+* Mobile application: `http://localhost:9002/?url=http://localhost:8001#!/workorders/list`
```
Comment (wtrocki) [inline docs/workforce-management-framework/topics/pro-running-the-demo-repositories.adoc]: `http://localhost:9002/?url=http://localhost:8001`

File: docs/workforce-management-framework/topics/pro-running-the-demo-repositories.adoc [hunk 3]
```diff
@@ -67,16 +67,29 @@ NOTE: The terminal is out of use while the Core Server is running.
...
+[source,bash]
+----
+cd raincatcher-mobile/
+npm run start
+----
+
+The Portal and Mobile applications automatically open in a browser at the
+following URLs:
+
+* Portal application: `http://localhost:8001/cookie-login`
```
Comment (wtrocki) [inline docs/workforce-management-framework/topics/pro-running-the-demo-repositories.adoc]: `http://localhost:9003/?url=http://localhost:8001`

File: docs/workforce-management-framework/topics/pro-running-the-demo-repositories.adoc [hunk 4]
```diff
@@ -67,16 +67,29 @@ NOTE: The terminal is out of use while the Core Server is running.
...
-Both the Mobile and Portal Applications open automatically in a browser.
+2&#46; Start the Mobile application as follows:
+
+[source,bash]
+----
+cd raincatcher-mobile/
+npm run start
+----
+
+The Portal and Mobile applications automatically open in a browser at the
```
Comment (wtrocki) [inline docs/workforce-management-framework/topics/pro-running-the-demo-repositories.adoc]: IMHO Let's rephrase this that they will open automatically, but if not see bellow.

File: docs/workforce-management-framework/topics/pro-running-the-demo-repositories.adoc (hunk 1 above)
Comment (oraNod) [reply docs/workforce-management-framework/topics/pro-running-the-demo-repositories.adoc]: yeah, this is kind of a hack that I don't like. it renders as 2.

the reason for this is that if you insert [source,bash] between steps in an ordered list, or any other code snippet or content, it breaks the numbering when the list is rendered. it starts numbering over. so what should be step 2 becomes step 1.

my solution is to 'hard-code' the numbering and force it by inserting that entity.

if you don't want to keep this, and that's fine, perhaps a solution would be to say "First, start the..." and then "Next, start the..."

**Review by wtrocki — APPROVED** [review]

**Pull request merged**